frontend/utility/BasicOutputHandler.cpp

```diff
@@ -0,0 +1,584 @@
+#include "BasicOutputHandler.hpp"
+#include "AdvancedOutput.hpp"
+#include "SimpleOutput.hpp"
+
+#include <utility/MultitrackVideoError.hpp>
+#include <utility/StartMultiTrackVideoStreamingGuard.hpp>
+#include <utility/VCamConfig.hpp>
+#include <widgets/OBSBasic.hpp>
+
+#include <qt-wrappers.hpp>
+
+#include <QThreadPool>
+
+using namespace std;
+
+extern bool EncoderAvailable(const char *encoder);
+
+volatile bool streaming_active = false;
+volatile bool recording_active = false;
+volatile bool recording_paused = false;
+volatile bool replaybuf_active = false;
+volatile bool virtualcam_active = false;
+
+void OBSStreamStarting(void *data, calldata_t *params)
```

Comment:
Various functions here had `static` removed. Was that intentional?

Reply:
Correct - the functions are used by the simple and advanced output variants, so they cannot be `static` anymore.